The path for the default font used by VLC is debian specific it seems.
Fix the path and add DejaVU to the dependencies.

Index: vlc-master/modules/visualization/projectm.cpp
===================================================================
--- vlc-master.orig/modules/visualization/projectm.cpp
+++ vlc-master/modules/visualization/projectm.cpp
@@ -78,8 +78,8 @@ static void Close        ( vlc_object_t
 # define FONT_PATH      "C:\\WINDOWS\\Fonts\\arial.ttf"
 # define FONT_PATH_MENU "C:\\WINDOWS\\Fonts\\arial.ttf"
 #else
-# define FONT_PATH      "/usr/share/fonts/truetype/ttf-dejavu/DejaVuSans.ttf"
-# define FONT_PATH_MENU "/usr/share/fonts/truetype/ttf-dejavu/DejaVuSansMono.ttf"
+# define FONT_PATH      "/usr/share/fonts/dejavu/DejaVuSans.ttf"
+# define FONT_PATH_MENU "/usr/share/fonts/dejavu/DejaVuSansMono.ttf"
 #endif
 
 vlc_module_begin ()
